test/slow/api_spec_slow.js

```diff
@@ -195,20 +205,25 @@ describe(`electron-forge API (with installer=${installer.substr(12)})`, () => {
         targets = fs.readdirSync(path.resolve(__dirname, `../../src/makers/${process.platform}`)).map(file => path.parse(file).name);
       }
       const genericTargets = fs.readdirSync(path.resolve(__dirname, '../../src/makers/generic')).map(file => path.parse(file).name);
-
-      [].concat(targets).concat(genericTargets).forEach((target) => {
-        describe(`make (with target=${target})`, () => {
-          before(async () => {
+      const testMakeTarget = async function testMakeTarget(target, options = {}) {
+        await describe(`make (with target=${target})`, async () => {
```

Comment:
You can't await a describe 👍 

Calling describe doesn't isntantly run the test, it registers the suite to be run in sequence 👍